### ISSUE TITLE
[AdaptiveCards- Samples - Activity Update]: Tooltip is not defined for 'Date-picker' button under Activity Update.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8355]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/samples/ActivityUpdate.html
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)

**Repro Steps:**
1. Open the above URL with valid credential.
2. Navigate to 'Samples' link and press tab key to activate it.
3. Select the 'Activity Update' option and navigate to 'Date Picker' under 'Publish Adaptive Card Schema'.
4. Navigate to 'Date Picker' and 'Time Picker' option and press ctrl+shift+F10 and observe the issue.

**Actual Result:**
Tooltip is not defined for the 'Date Picker' and 'Time Picker' button, respectively.

**Expected Result:**
Tooltip should be defined for the 'Date Picker' and 'Time Picker' button, respectively.

**Note:**
1. Same issue is observed with 'Next Month' and 'Previous month' button as well.
2. Same issue is repro throughout the application.

**User Experience:**
Users who rely on AT will face difficulty if the icon button doesn't have tooltip defined, users may not know what the purpose of the element if no tooltip is present.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="956" alt="Usable-Tooltip is not defined for the Date Picker and time picker button" src="https://user-images.githubusercontent.com/95913748/224231121-ab994466-7f3d-4192-a015-11f30aa2a106.png">

### Card JSON

```JSON
NA
```
